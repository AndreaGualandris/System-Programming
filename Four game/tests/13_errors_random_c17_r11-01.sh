#!/bin/bash
exec "$1" 7 5 o < tests/10_random_c17_r11-01.txt
