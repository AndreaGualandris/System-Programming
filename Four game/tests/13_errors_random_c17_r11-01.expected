-------
       
       
       
       
       
-------
0123456
Player 'o' to move:
  o
  v
-------
       
       
       
       
  o    
-------
0123456
Player 'x' to move:
Illegal move.
Illegal move.
  x
  v
-------
       
       
       
  x    
  o    
-------
0123456
Player 'o' to move:
   o
   v
-------
       
       
       
  x    
  oo   
-------
0123456
Player 'x' to move:
   x
   v
-------
       
       
       
  xx   
  oo   
-------
0123456
Player 'o' to move:
Illegal move.
Illegal move.
Illegal move.
Illegal move.
   o
   v
-------
       
       
   o   
  xx   
  oo   
-------
0123456
Player 'x' to move:
Illegal move.
 x
 v
-------
       
       
   o   
  xx   
 xoo   
-------
0123456
Player 'o' to move:
Illegal move.
Illegal move.
    o
    v
-------
       
       
   o   
  xx   
 xooo  
-------
0123456
Player 'x' to move:
Illegal move.
Illegal move.
Illegal move.
Illegal move.
Illegal move.
Illegal move.
 x
 v
-------
       
       
   o   
 xxx   
 xooo  
-------
0123456
Player 'o' to move:
Illegal move.
  o
  v
-------
       
       
  oo   
 xxx   
 xooo  
-------
0123456
Player 'x' to move:
Illegal move.
 x
 v
-------
       
       
 xoo   
 xxx   
 xooo  
-------
0123456
Player 'o' to move:
Illegal move.
Illegal move.
Illegal move.
Illegal move.
 o
 v
-------
       
 o     
 xoo   
 xxx   
 xooo  
-------
0123456
Player 'x' to move:
Illegal move.
Illegal move.
Illegal move.
Illegal move.
  x
  v
-------
       
 ox    
 xoo   
 xxx   
 xooo  
-------
0123456
Player 'o' to move:
Illegal move.
Illegal move.
     o
     v
-------
       
 ox    
 xoo   
 xxx   
 xoooo 
-------
0123456
Player 'o' wins!
